packages/hub/package.json

```diff
@@ -9,14 +9,14 @@
   "scripts": {
     "db:drop": "dropdb hub_wallet_${NODE_ENV}",
     "db:create": "createdb hub_wallet_${NODE_ENV}",
-    "db:migrate": "npx knex migrate:latest --knexfile lib/config/knexfile",
-    "db:rollback": "npx knex migrate:rollback --knexfile lib/config/knexfile",
-    "db:seed": "npx knex seed:run --knexfile lib/config/knexfile",
-    "hub:start:dev": "node ./lib/start-dev-server.js",
+    "db:migrate": "npx knex migrate:latest --knexfile lib/db-admin/knexfile",
```

Comment:
I'd suggest adding a higher-level script like `db:setup` that'd run all of these as I expect in a development flow you're more often running these commands back to back instead of in isolation.

Though I could be wrong as @kerzhner has been iterating more heavily on the hub recently.

Reply:
That's a great idea! TBH, I haven't really been running the hub much recently. I have been relying mostly on unit tests. And unit tests do not use these yarn commands.